composer.json

```diff
@@ -11,22 +11,22 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
-        "php-http/httplug": "^2.0",
+        "php": "^8.1",
+        "php-http/httplug": "^2.4",
         "php-http/message": "^1.6",
         "psr/http-client": "^1.0",
         "psr/http-factory": "^1.0",
-        "psr/http-message": "^1.0 || ^2.0",
-        "symfony/options-resolver": "~4.0.15 || ~4.1.9 || ^4.2.1 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/polyfill-php80": "^1.17"
+        "psr/http-message": "^2.0",
+        "symfony/options-resolver": "^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "doctrine/instantiator": "^1.1",
-        "guzzlehttp/psr7": "^1.4",
-        "nyholm/psr7": "^1.2",
-        "phpspec/phpspec": "^5.1 || ^6.3 || ^7.1",
-        "phpspec/prophecy": "^1.10.2",
-        "phpunit/phpunit": "^7.5.20 || ^8.5.33 || ^9.6.7"
+        "doctrine/instantiator": "^2.0",
```

Comment:
do we start to conflict with instantiator 1 or is that simply not installable with our php / symfony versions?

Reply:
I do not know. But it is a dev-dependency. So i usally do not care about older versions

Reply:
do we need to depend on instantiator at all?

if it is an optional dependency of us, i worry that if we are no longer compatible with 1, people could install us with instantiator 1 and end up with broken code.

if we actually need the dependency and 1.x no longer works, we should mark < 2.0 as conflict.

Reply:
I removed it. Looks like it it not needed.

Reply:
thanks. 
maybe it was added at some point to prevent issues with lowest build, or something :shrug: